lib/elixir_internal_certificate_web/controllers/api/v1/user_search_controller.ex

```diff
@@ -54,6 +55,43 @@ defmodule ElixirInternalCertificateWeb.Api.V1.UserSearchController do
     end
   end
 
+  def upload(
```

Comment:
should we use `create` method 🤔

Reply:
Updated in [744af57](https://github.com/nvminhtue/elixir_internal_certificate/pull/47/commits/744af573ac7479b456b911010465ce261483f7cc)